web/src/styles/onboarding.css

```
@@ -385,35 +661,142 @@
+  border: none;
+  border-radius: 6px;
+  background: transparent;
+  color: var(--text-secondary);
+  cursor: pointer;
+  transition: background 0.15s, color 0.15s, text-shadow 0.15s;
+}
+
+.key-tab.active {
+  background: var(--bg-card, #fff);
+  color: var(--text);
+  text-shadow: 0 0 0.01px currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)